assets/js/components/Dashboard/AlertBanner.tsx

```diff
@@ -1,17 +1,27 @@
 import React, { ComponentType } from "react";
-import { ArrowRepeat } from "react-bootstrap-icons";
+import { ArrowRepeat, CheckCircleFill } from "react-bootstrap-icons";
 import { formatEffect, translateRouteID } from "../../util";
 import { useParams } from "react-router-dom";
-import { useScreenplayContext } from "../../hooks/useScreenplayContext";
+import {
+  useScreenplayContext,
+  useScreenplayDispatchContext,
+} from "../../hooks/useScreenplayContext";
 import { Alert } from "../../models/alert";
 
 interface BannerAlert {
   alert: Alert;
   closedAt?: string;
 }
 
-const AlertBanner: ComponentType = () => {
+interface AlertBannerProps {
+  isDone: boolean;
+}
+
+const AlertBanner: ComponentType<AlertBannerProps> = ({
+  isDone,
+}: AlertBannerProps) => {
   const { bannerAlert } = useScreenplayContext();
+  const dispatch = useScreenplayDispatchContext();
```

Comment:
Hmm, good question. My thinking there is as long as a component has access to the Context, it can talk directly to it. But this kind of thing is normally handled with a function in props, so I'm torn. I don't personally have a strong opinion one way or the other so I'm happy to change it or take it up in the eng Slack channel and see what others think.

Reply:
Given the change we made in the "expand place rows" PR, is that an extra + for using callback function props? Can certainly take it up in the eng Slack

Reply:
Yep, I think so. Went ahead and changed it here.